tutorials/dapt-curation/code/main.py

```diff
@@ -234,11 +268,13 @@ def main():
     args = ArgumentHelper(parser).add_distributed_args().parse_args()
     # Limit the total number of workers to ensure we don't run out of memory.
     args.n_workers = min(args.n_workers, 8)
+    args.device = "gpu"
```

Comment:
Instead of hardcoding this, we need to let the users decide what device to use. Then under the hood, ensure that GPU processing steps only run if the user has passed `--device gpu`